src/content/pages/attend/_index.md

```diff
@@ -1,10 +1,9 @@
 ---
 title: Attend PyCon AU
-order: 1
 ---
 
 # More details coming soon!
 
 We'll have more information about how you can attend PyCon AU coming soon.
 
-In the meantime, If you would like to apply for a volunteering position at this year's conference, please [visit the volunteers page](/attend/volunteer) for more information.
+In the meantime, If you would like to apply for a volunteering position at this year's conference, please [visit the volunteers page](/attend/volunteer) for more information.
```

Comment:
In the meantime, **I**f you would <-- probably lower case `I` in `if`

Reply:
I've made that change, but the page has been hidden from view anyway (the thing i changed was adding a `_` to the filename, effectively unpublishing it.